client/src/partials/stock/exit/distribution.js

```diff
@@ -67,9 +67,8 @@ angular.module('bhima.controllers')
 
     function startup(model) {
       angular.extend($scope, model);
-
+      $scope.ledger.data = $scope.ledger.data.filter(function (data) { return data.is_distributable[0] === 1; });
```

Comment:
The *function debtor* in **ledger.js** is used in more than one place, if i change the query clause 'where' it's will affect other place in the application.